snap/snapcraft.yaml

```diff
@@ -1,5 +1,5 @@
 name: kata-containers
-version: @KATA_RUNTIME_VERSION@
+version: 1.5.0-rc2
```

Comment:
- This looks wrong to me as it isn't being quoted - see https://github.com/kata-containers/runtime/blob/master/versions.yaml#L52..L55.

- This appears to be the only version that isn't being queried dynamically, but it's an important one ;)

  I'm not 100% clear on why this PR is needed. It sounds like we cannot run `make` on the launchpad infrastructure (which would allow this version number to be generated), but I'm not clear why that is a problem *iff* the resulting `snapcraft.yaml` is uploaded to the launchpad site (just run `make` locally and upload). Is this PR designed to avoid any manual uploading so that the snap build can be handled entirely by the launchpad site?

Reply:
> This looks wrong to me as it isn't being quoted - see https://github.com/kata-containers/runtime/blob/master/versions.yaml#L52..L55.

according with the doc https://docs.snapcraft.io/the-snap-format/698, quotes are not required

> I'm not 100% clear on why this PR is needed ...

to keep it simple and easy to maintain, I prefer to create a repo in launchpad (https://launchpad.net/~kata-containers-dev) pointing to this repo and just bump the snap version, launchpad can automatically pull this repo and build & push (not release) snaps in the store

Reply:
> according with the doc https://docs.snapcraft.io/the-snap-format/698, quotes are not required

But all the examples in that doc use "floating point version numbers" (`a.b`) whereas we're using semver (`a.b.c.d`). I haven't checked the YAML spec, but I still think that for our versioning, quoting would be safer.

>  launchpad can automatically pull this repo

How is the pull triggered? Do you need to hit a button somewhere, or is launchpad monitoring the commits in this repo?

Reply:
> But all the examples in that doc use "floating point version numbers" (a.b) whereas we're using semver (a.b.c.d). I haven't checked the YAML spec, but I still think that for our versioning, quoting would be safer.

agree

> How is the pull triggered? Do you need to hit a button somewhere, or is launchpad monitoring the commits in this repo?

![screenshot from 2019-01-11 10-55-15](https://user-images.githubusercontent.com/13009432/51047958-675a7980-158f-11e9-9bed-a9ed1d4d9fe4.png)

and

![screenshot from 2019-01-11 10-53-49](https://user-images.githubusercontent.com/13009432/51047977-6cb7c400-158f-11e9-8028-9b64467962a8.png)